docs/topics/yield.rst

```diff
@@ -0,0 +1,176 @@
+.. _topics-yield:
+
+==========================
+'Yield' Keyword in Python and its use for Scrapy Callbacks
+==========================
+  
+What is ``yield``?
```

Comment:
I don't think we need such a long explanation what are Python generator and how does the yield keyword in Python work, there are better existing places for that.